ProboCI/eslint-config-probo
# Update eslint syntax.

Several of our checks have been deprecated in favor of a more
modern syntax.
- http://eslint.org/docs/rules/no-empty-label
- http://eslint.org/docs/rules/space-after-keywords
- http://eslint.org/docs/rules/space-return-throw-case

This updates our config to support the most recent version.
### To test
- Download into a project that is being linted using these rules and ensure no new errors or warnings are displayed.
